internal/yaml/yaml.go

```diff
@@ -11,6 +11,12 @@ const (
 	maxDecodedValues = 1000000
 )
 
+// MapSlice is aliased from yaml.MapSlice
+type MapSlice = yaml.MapSlice
```

Comment:
Why aliasing this type? To avoid to import a second yaml package in yatee and then avoid the collision problem? You could also import the external yaml and rename our internal when importing
```go
import(
    ...
    yml "github.com/docker/app/internal/yaml"
    "gopkg.in/yaml.v2"
    ...
)